src/pages/home/categories-list/tests/categories-list.spec.js

```diff
@@ -22,14 +24,27 @@ useQuery.mockImplementation(() => ({
   error: false
 }));
 
-describe('tests for categories list', () => {
-  const wrapper = mount(
-    <CategoriesContextProvider>
-      <CategoriesList />
-    </CategoriesContextProvider>
-  );
+describe('tests for categories list component', () => {
+  beforeEach(() => {
+    render(
+      <CategoriesContextProvider>
+        <CategoriesList />
+      </CategoriesContextProvider>
+    );
+  });
+
+  it('should exist with carousel wrapper', () => {
+    const title = screen.getByRole('heading', { level: 2 });
+    const list = screen.getByRole('list');
+
+    expect(title).toBeInTheDocument();
+    expect(list).toBeInTheDocument();
+  });
+
+  it('getCategoryURL function should return correct string', () => {
+    const res = getCategoryURL(mockCategory);
+    const etalonString = 'catalog/test';
```

Comment:
I think it is better to remove it to variables file
You also need to delete those tests from this folder
and create new files in src/tests/unit/..

Reply:
Ok